ddcz/views/users.py

```diff
@@ -117,3 +122,34 @@ def change_skin(request):
         redirect_url = "/"
 
     return HttpResponseRedirect(redirect_url)
+
+
+@require_http_methods(["HEAD", "GET", "POST"])
+def awaiting(request):
+    registrations = UzivateleCekajici.objects.all().order_by("-date")
+    awaiting = registrations.count() is not 0
+
+    if request.method == "POST":
```

Comment:
Combining those two lines?

Reply:
Probably not, I am sending registrations into the template. I think it should be two different sql queries in the end. Line 150